### PR TITLE
MNEMONIC-703: Add redis memory service in mnemonic project

### PR DESCRIPTION
**Pull request by yzz127**
For redis integration into mnemonic as backend storage, we need to add the redis memory service to the project

### CODE REVIEW

File: mnemonic-memory-services/mnemonic-redis-service/build.gradle [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+description = 'mnemonic-redis-service'
+
+dependencies {
+  implementation project(':mnemonic-common')
+  testCompileOnly 'org.testng:testng'
+}
+
+shadowJar {
```
Comment (bigdata-memory) [inline mnemonic-memory-services/mnemonic-redis-service/build.gradle:30]: I think it is obsoleted, and sugguest creating another ticket to remove them all, thanks.

**Review by bigdata-memory — APPROVED** [review]
It is goood as a skeleton code for Redis memory service integration, thanks.

File: mnemonic-memory-services/mnemonic-redis-service/build.gradle (hunk 1 above)
Comment (yzz127) [reply mnemonic-memory-services/mnemonic-redis-service/build.gradle:30]: Sure, thanks for the input!

**Review by jxjlearn — APPROVED** [review]
Looks good, thanks!

**Pull request merged**